src/main/java/qna/domain/DeleteHistory.java

```diff
@@ -23,13 +23,21 @@ public class DeleteHistory {
 
     protected DeleteHistory() {}
 
-    public DeleteHistory(ContentType contentType, Long contentId, User deletedBy, LocalDateTime createDate) {
+    private DeleteHistory(ContentType contentType, Long contentId, User deletedBy, LocalDateTime createDate) {
         this.contentType = contentType;
         this.contentId = contentId;
         this.deletedBy = deletedBy;
         this.createDate = createDate;
     }
 
+    public static DeleteHistory ofQuestion(Long questionId, User deletedBy, LocalDateTime createDate) {
+        return new DeleteHistory(ContentType.QUESTION, questionId, deletedBy, createDate);
+    }
```

Comment:
LocalDateTime.now() 를 매개변수로 받지 않아도 좋을 것 같아요 😄 
```suggestion
    public static DeleteHistory ofQuestion(Long questionId, User deletedBy) {
        return new DeleteHistory(ContentType.QUESTION, questionId, deletedBy, LocalDateTime.now());
    }
```